microsoft/electionguard-api-python
# 📫 #221 Postman Collection Tests for Election Ceremony

### Issue

Fixes #221 

### Description
Includes all requests to complete a key ceremony and produce an elgamal public key for an election with two guardians and a quorum of two.

### Testing
1. Import the postman collection: `ElectionGuard API Tests.postman_collection.json`
2. Import the Environment: `Local Environment.postman_environment.json`
3. If you want manually run through the authenticate and login steps, then all 19 steps of the key ceremony:

![image](https://user-images.githubusercontent.com/1769905/158805428-88e69950-a5e2-43b1-852e-ba654489bd11.png)

4. Run through the tests automatically by clicking "End-To-End Happy Path" and "Run" in the top right

Expected: all tests pass and look like this:

![image](https://user-images.githubusercontent.com/1769905/158816506-008f1577-2645-4b3a-8ab6-3acacb81d148.png)

Expected: The last test 3.19 will print the name of the key ceremony
